lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1896,19 +1898,13 @@ export class UserAgentApplication {
    */
   // TODO: Break this function up - either into utils or token specific --- too long to be readable
   protected saveTokenFromHash(tokenResponse: TokenResponse): void {
+
     this._logger.info("State status:" + tokenResponse.stateMatch + "; Request type:" + tokenResponse.requestType);
     this._cacheStorage.setItem(Constants.msalError, "");
     this._cacheStorage.setItem(Constants.msalErrorDescription, "");
-    var scope: string = "";
-    var authorityKey: string = "";
-    var acquireTokenUserKey: string = "";
-    // if response has "scope" set it, else set the scope to client_id
-    if (tokenResponse.parameters.hasOwnProperty("scope")) {
```

Comment:
why did you remove this? did I miss something?

Reply:
Thanks for catching this. I think it is an oversight while merging. Added the code back. Will commit soon.

Reply:
Actually may be not. Sorry, let me check. I think scope was never used. Give me some time, will update.

Reply:
@manoj-rath 'scope' in saveTokenFromHash is never used. Hence removed the un-used code fragment.

Reply:
I also couldn't see scope being used in this method, however I know that this logic "if response has "scope" set it, else set the scope to client_id" exists. So I am not sure if the scope being "var" (global scope) is making it work. please double check if it works correctly by removing this code.

Reply:
I checked the code again and here is what is happening: 
saveTokenFromHash() is the function that has the above code which processes the response from the server, this internally calls saveAccessToken() which has the above logic of "if response has "scope" set it, else set the scope to client_id". 

The deleted code is just redundant and never used.
I think it is safe to remove this.

@pkanher617 can you also check and confirm to ensure no mistakes?